pkg/compute/rust.go

```diff
@@ -153,9 +153,13 @@ func (r Rust) Verify(out io.Writer) error {
 		return fmt.Errorf("error parsing rustup constraint: %w", err)
 	}
 	if !rustupConstraint.Check(rustupVersion) {
+		pre := "To fix this error, run the following command"
+		cmd := text.Bold("rustup self update")
+		alt := fmt.Sprintf("%s If you installed rustup using a package manager, refer to its documentation to update it.", text.Bold("INFO:"))
```

Comment:
perhaps "package manager such as Homebrew"?

Reply:
I had hoped to avoid calling out specific package managers (although admittedly the issue we're addressing has been reported because of Homebrew).

Reply:
What are we trying to accomplish by naming a specific package manager?

Personally I'd prefer if we don't, as this might be the case with any of them and/or might change in the future without our knowledge.

Reply:
I'm less worried about that and more about implying that we want the user to update their package manager 😂 .  I might say don't mention homebrew unless we can be sure that we're only doing that on MacOS platforms.

```suggestion
		alt := fmt.Sprintf("%s If you installed rustup using a package manager, you may need to follow your package manager's documentation to update the rustup package.", text.Bold("INFO:"))
```